code/modules/response_team/ert_outfits.dm

```diff
@@ -499,15 +499,15 @@
 		/obj/item/grenade/chem_grenade/antiweed = 2,
 		/obj/item/lightreplacer = 1,
 		/obj/item/storage/bag/trash = 1,
-		/obj/item/push_broom,
+		/obj/item/push_broom = 1,
 		/obj/item/storage/box/lights/mixed = 1,
 		/obj/item/melee/flyswatter = 1)
 
 /datum/outfit/job/centcom/response_team/janitorial/amber
 	name = "RT Janitor (Amber)"
 	suit = /obj/item/clothing/suit/armor/vest/ert/janitor
 	head = /obj/item/clothing/head/helmet/ert/janitor
-	glasses = /obj/item/clothing/glasses/sunglasses
+	glasses = /obj/item/clothing/glasses/hud/janitor/sunglasses
```

Comment:
Could you please give all these guys sec-hud implants?

Reply:
Amber, red and gamma janitors? We want all ERT loadouts to have access to sec hud?

Reply:
Yeah, I'd say all of them just to not make this an ERT balance change

Reply:
Amber = sunglasses
Red = sec sunglasses
Gamma = nightvision goggles
Thats what they had, giving them all sec-hud implants would be an ERT balance change.

Reply:
Eh you're right, just give it to what ones had it before then I guess

Reply:
Done in next commit (why is gamma engineer the only non sec themed ERT that gets a sec-hud implant????)